flags.css

```diff
@@ -1,8 +1,56 @@
 /* Indonesia */
+.indonesiaFlag {
+    border: 2px solid;
+    height: 200px;
+    width: 300px;
+}
+.indonesiaStrip1 {
+    background-color: red;
+    height: 100px;
+}
 
 /* Colombia */
-
+.columbiaFlag {
+    margin-top: 50px;
+    border: 2px solid;
+    height: 25vh;
+    width: 300px;
+}
+.columbiaStrip1 {
+    background-color: gold;
+    height: 50%;
+}
+.columbiaStrip2,
+.columbiaStrip3 {
+    height: 25%;
+}
+.columbiaStrip2 {
+    background-color: mediumblue;
+}
+.columbiaStrip3 {
+    background-color: firebrick;
+}
 /* Ireland */
+.irelandFlag {
+    margin-top: 50px;
+    border: 2px solid;
+    height: 25vh;
+    width: 300px;
```

Comment:
see how you have to keep setting this each time?  if you had done it on the flagContainer class, you would have only had to set it once.  Then give something like switzerland a different setting because the swiss are special